neutron/files/mitaka/neutron-server.conf.Debian

```diff
@@ -28,14 +28,19 @@ bind_port = {{ server.bind.port }}
 
 {% if server.backend.engine == "contrail" %}
 
-# TEMPORARY - until neutron v2 contrail package would be supported
-#api_extensions_path = extensions:/usr/lib/python2.7/dist-packages/neutron_plugin_contrail/extensions:/usr/lib/python2.7/dist-packages/neutron_lbaas/extensions
+{% if server.get('lbaas', {}).get('providers', {}).get('Opencontrail', {}).get('engine', 'contrail') == 'contrail-lbaasv2' %}
+api_extensions_path = extensions:/usr/lib/python2.7/dist-packages/neutron_plugin_contrail/extensions:/usr/lib/python2.7/dist-packages/neutron_lbaas/extensions
+service_plugins = neutron_plugin_contrail.plugins.opencontrail.loadbalancer.v2.plugin.LoadBalancerPluginV2
```

Comment:
pls. make shure, (post confirmation) the plugin paths works with juniper distribution packaging as well.